### ISSUE TITLE
telemetry howto

### ISSUE THREAD

**Issue by mogyoros** [librespeed/speedtest-cli#2]
Hi,
I have successfully managed to compile the source code with build.sh, and run the program with a local LibreSpeed server instance.
```
 # ./librespeed-cli-linux-amd64 --simple --local-json server.json  --telemetry-json telemetry.json
Ping:   0 ms    Jitter: 0 ms
Download rate:  4959.95 Mbps
Upload rate:    2572.16 Mbps

```
Content of local server.json file:
```
[
  {
    "name": "PHP Backend",
    "server": "http://localhost:84/backend",
    "dlURL": "garbage.php",
    "ulURL": "empty.php",
    "pingURL": "empty.php",
    "getIpURL": "getIP.php"
  }
]
```
Content of local telemetry.json:
```
  {
    "telemetryLevel": "full",
    "server": "http://localhost:84",
    "path": "/results/telemetry.php",
    "shareURL": "/results/"
  }
```
But the program does not upload the results into the specified telemetry server.
I captured the communication, but I don't see any HTTP post req in the end.
Maybe this is not implemented yet?
Kind regards,
Tamas

**Comment by maddie** [comment]
Hi,

You have to use `--share` option to enable telemetry uploading.

**Comment by mogyoros** [comment]
Hi,
Thanks for the hint! :-)
I would not find this out myself from the documentation. Maybe you could mention it in the "Use a custom telemetry server" section, to make it clear,
Thanks again!
Tamas

**Comment by maddie** [comment]
Thanks for your suggestions, I'll try to improve the documentation.

**Comment by Perflyst** [comment]
`--telemetry-json` without `--share` is useless
maybe `--telemetry-json` should automatically enable sharing?

**Comment by maddie** [comment]
@Perflyst Yes, I think that would be more reasonable.

**Comment by maddie** [comment]
Fixed in commit e61ed2c, will include in next release.
